web_fragments/tests/test_fragment.py

```diff
@@ -13,19 +13,9 @@
 from django.test import TestCase
 
 from web_fragments.fragment import Fragment, FragmentResource
-from web_fragments.test_utils import (
-    CSS_EXPECTED_HTML,
-    CSS_LINK_EXPECTED_HTML,
-    JS_EXPECTED_HTML,
-    JS_LINK_EXPECTED_HTML,
-    TEST_CSS,
-    TEST_CSS_URL,
-    TEST_HTML,
-    TEST_JS,
-    TEST_JS_INIT_FN,
-    TEST_JS_URL,
-    TEST_JSON_INIT_ARGS
-)
+from web_fragments.test_utils import (CSS_EXPECTED_HTML, CSS_LINK_EXPECTED_HTML, JS_EXPECTED_HTML,
+                                      JS_LINK_EXPECTED_HTML, TEST_CSS, TEST_CSS_URL, TEST_HTML, TEST_JS,
+                                      TEST_JS_INIT_FN, TEST_JS_URL, TEST_JSON_INIT_ARGS)
```

Comment:
I'm curious, what prompted this change?  We generally find the one-item-per-line formatting more readable in cases like this.

Reply:
isort did that and I thought best to keep it.

Reply:
Hmm, could you please add `multi_line_output=3` to the isort configuration in setup.cfg to fix this?  We [do this in edx-platform](https://github.com/edx/edx-platform/blob/master/setup.cfg#L39); explanation for the setting is [here](https://github.com/timothycrosley/isort#multi-line-output-modes).